package.json

```diff
@@ -89,6 +89,7 @@
     "@testing-library/react-hooks": "^7.0.0",
     "@types/debug": "^4.1.5",
     "@types/file-saver": "^2.0.3",
+    "@types/gtag.js": "0.0.8",
```

Comment:
why pin the dependency though?

Reply:
strange, re-tried fresh install it auto-pin still. nvm i manually put caret